lib/module_importer.js

```diff
@@ -105,24 +103,15 @@ function makeImporter(eyeglass, sass, options, fallbackImporter) {
 
   return function(uri, prev, done) {
     var isRealFile = fs.existsSync(prev);
-    var match = uri.match(IMPORT_REGEX);
-
-    if (match) {
-      // This is an import of the form "<node_module>/some/file"
-      var moduleName = match[1];
-      var relativePath = match[2];
-      var pkgRootDir = isRealFile ? packageRootDir(path.dirname(prev)) : root;
-      var jsFile = getModuleByName(moduleName, pkgRootDir);
-
-      if (!jsFile) {
-        console.error("No Eyeglass module named '" + moduleName +
```

Comment:
@Jakobo shouldn't we be able to show this error in the last case of this [if-else block](https://github.com/sass-eyeglass/eyeglass/blob/master/lib/module_importer.js#L167)?